client/components/foldable-card/style.scss

```diff
@@ -134,7 +134,7 @@ button.foldable-card__action {
 .foldable-card.is-expanded .foldable-card__content {
 	display: block;
 	padding: 16px;
-	border-top: 1px solid var( --color-neutral-0 );
+	border-top: 1px solid var( --color-neutral-50 );
```

Comment:
Is it international to target the whole component here? This would affect the border of all places the component is used throughout Calypso, for example, the exporter. 

Though that being said, it may be worth that being the case for a more consistent experience where the border is easier to see in all scenarios.

Reply:
Good point. I think it's worth updating the component so this change is applied globally. But I appreciate that might be subjective. For me, the current stroke color is so light it's hardly visible. Here's a before / after of the exporter:

Before:

![before](https://cldup.com/6zXEqoUG3d-3000x3000.png)

After: 

![after](https://cldup.com/TBWh-TD_HO-3000x3000.png)

Reply:
I would argue that if you're going to make this change, of which i have no objection to, it would be good to make it to the Card component as well for consistency. What do you think?

Reply:
@MichaelArestad how do you mean? Make the Card border match the border color update in this PR? I haven't inspected the actual values, but I _think_ that's what is happening?

Reply:
Ah! If that's the case, then we're all good. I just wanted everything the same for consistency.